File: .ProgramingBasics/SimpleConditionalStatements/01.Excellent Result/src/ExcellentResult.java
import java.util.Scanner;

public class ExcellentResult {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double grade = Double.parseDouble(scanner.nextLine());

        if (grade >= 5.50) {
            System.out.println("Excellent!");
        }
    }
}

File: ProgramingBasics/SimpleConditionalStatements/01.ExcellentResult/01.ExcellentResult/ExcellentResult.cs
using System;

namespace _01.ExcellentResult
{
    public class ExcellentResult
    {
        public static void Main(string[] args)
        {
            var grade = double.Parse(Console.ReadLine());

            if (grade >= 5.50)
            {
                Console.WriteLine("Excellent!");
            }
        }
    }
}
